man/man4/zfs.4

```diff
@@ -1354,6 +1354,22 @@ _
 .TE
 .Sy \& * No Requires debug build.
 .
+.It Sy zfs_btree_verify_intensity Ns = Ns Sy 0 Pq uint
+Enables btree verification.
+The following settings are culminative:
```

Comment:
If we make this user visible, let's make it clear that levels 4 and 5 are expensive.

Reply:
Agreed. I will change this in my next revision tomorrow morning.

That said, level 4 is the one that particularly interests me as something that we might want to ask certain end users to try running.

Reply:
I have added `(expensive)` to the end of the descriptions of levels 4 and 5 to indicate that they are expensive.